README.md

```diff
@@ -49,6 +49,7 @@ This README contains some high level overview information on Atomic App. The det
 8. [Specification coverage](docs/spec_coverage.md)
```

Comment:
should we rip out anything in the spec that isn't supported by atomicapp and then list those features in a TODO list somewhere. One of the benefits of putting these two together would be to reduce confusion IMHO. Having them out of sync in the same repo wouldn't make much sense.